tests/legacy-e2e/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceDetails.java

```diff
@@ -57,8 +57,7 @@
     String APPLY_CHANGES_BUTTON = "//che-button-save-flat[@class='apply-button']";
     String CANCEL_CHANGES_BUTTON = "//button[@name='cancel-button']";
     String CANCEL_DIALOG_BUTTON = "//md-dialog[@role='dialog']//button/span[text()='Cancel']";
-    String CLOSE_DIALOG_BUTTON =
-        "//*[@id='cancel-dialog-button']//*[contains(@class, 'che-button')]";
+    String CLOSE_DIALOG_BUTTON = "close-dialog-button";
```

Comment:
`CLOSE_DIALOG_BUTTON` > `CLOSE_DIALOG_BUTTON_ID` will make it clear how this constant should be applied.